tests/integration-tests/tests/storage/test_fsx_lustre/test_fsx_lustre_backup/pcluster.config.ini

```diff
@@ -0,0 +1,40 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+{% if scheduler == "awsbatch" %}
+min_vcpus = 4
+desired_vcpus = 4
+{% else %}
+initial_queue_size = 1
+maintain_initial_size = true
+{% endif %}
+fsx_settings = fsx
+
+[vpc parallelcluster-vpc]
+vpc_id = {{ vpc_id }}
+master_subnet_id = {{ public_subnet_id }}
+compute_subnet_id = {{ private_subnet_id }}
+use_public_ips = false
+
+[fsx fsx]
+shared_dir = {{ mount_dir }}
+storage_capacity = 1200
+weekly_maintenance_start_time = 1:00:00
+deployment_type = {{ deployment_type }}
+{% if per_unit_storage_throughput %}
+per_unit_storage_throughput = {{ per_unit_storage_throughput }}
+{% endif %}
+automatic_backup_retention_days = 2
```

Comment:
In my opinion, `deployment_type` and `per_unit_storage_throughput` should be hardcoded since the test doesn't validate anything based on these values.